File: 10.23研究生练习赛/a.py
# @Author: zjt
# @Date:   23-Oct-2020
# @Email:  jianting112358@gmail.com
# @Filename: a.py
# @Last modified by:   zjt
# @Last modified time: 24-Oct-2020
while True:
    try:
        n = map(int, raw_input().strip().split())
        i = int(n)
        str = " AC"
        print("AC"+(i-1)*str)
    except EOFError:
        break

File: 10.23研究生练习赛/a.cpp
/**
 * @Author: zjt
 * @Date:   23-Oct-2020
 * @Email:  jianting112358@gmail.com
 * @Filename: a.cpp
 * @Last modified by:   zjt
 * @Last modified time: 23-Oct-2020
 */
#include <bits/stdc++.h>
using namespace std;
typedef long long  LL;
typedef pair<int,int> PII;
#define mem(A, X) memset(A, X, sizeof A)
#define foreach(e,x) for(__typeof(x.begin()) e=x.begin();e!=x.end();++e)
#define fori(i,l,u) for(LL (i)=(LL)(l);(i)<=(LL)(u);++(i))
#define ford(i,l,u) for(LL (i)=(LL)(l);(i)>=(LL)(u);--(i))

int main()
{
  ios::sync_with_stdio(false);
  #ifdef LOCAL
  freopen("local.in","r",stdin);
  #endif
  int T;
  while(cin>>T){
      int n;
      while(T--){
          cin>>n;
          fori(i,1,n-1){
              cout<<"AC"<<" ";
          }
          cout<<"AC"<<endl;
      }
  }

return 0;
}
